internal/stage7.go

```diff
@@ -12,29 +13,34 @@ import (
 	"github.com/codecrafters-io/tester-utils/test_case_harness"
 )
 
-func getPath(executable string) string {
+func getPath(executable string, exePath string) string {
+	if executable == "my_exe" {
```

Comment:
@glokta1 this condition seems better placed within the caller - makes more sense with the context there. 

The name `exePath` is confusing too. 

One rule of thumb is that one should be able to guess what a function/variable does by looking at the signature and context (and not having to look at the implementation). 

If you were to look at this function: 

```
getPath(executable string) string
```

It's roughly clear what it does. 

Look at this instead: 

```
getPath(executable string, exePath string) string
```

Multiple questions can arise from this:

- Is `exePath` the path of the executable? If so, why are we passing it to `getPath` when we already have the path?